### PR TITLE
Gastropod Cooking Recipes

### PR DESCRIPTION
**Pull request by Xaritscin**
#### Summary
Content "adds two recipes for use with snails/slugs"

#### Purpose of change

This is something i have been wanting to add ever since i introduced mollusks to mainline. While there are basic recipes for gastropods there was no actual culinary use for these until now (i hope). this should give more use to all those harvested snails (and slugs) you may have found while foraging.

#### Describe the solution

-added Paella Valenciana (one of the original variations of this spanish dish), its a 2 portions dish based out of Deluxe Beans and Rice recipe with some adjustments
-added Escargot (cause it was expected to have this dish the moment we got snails for cooking), this dish is designed more like a snack in that it consists of 6 prepared "snails" with a bit more caloric push than the regular cooked gastropods due to the added butter and veggy components.

These recipes were included in the Family Cookbook but i think this may have to be moved in the future to some other cookbook.

#### Describe alternatives you've considered

let people just eat cooked gastropods with no culinary value

#### Testing

Generated a new world and tried the recipes, should be working as intended.

#### Additional context

I was thinking of including Mollusk Meat here but i designed that one to be like an alternative to squid, bivalves or octopus not straight out snails or slugs.

### CODE REVIEW

**Comment by Xaritscin** [comment]
after meddling around with the densities and caloric vales i think i finally got the thing working

File: data/json/items/comestibles/meat_dishes.json [hunk 1]
```diff
@@ -1671,5 +1671,47 @@
...
+    "id": "paella_valenciana",
+    "name": { "str_sp": "valencian paella" },
+    "conditional_names": [ { "type": "COMPONENT_ID", "condition": "mutant", "name": { "str_sp": "\"deluxe\" beans and rice" } } ],
+    "weight": "110 g",
+    "color": "brown",
+    "spoils_in": "15 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 442,
+    "description": "A traditional dish originated from Spain, more specifically from the region of Valencia.  A very tasty and filling dish made traditionally of rabbit, beans, rice, chicken, spices and sometimes snails.",
```
Comment (FoolsGold45) [inline data/json/items/comestibles/meat_dishes.json:1686]: "A traditional dish **originating** from Spain

Comment (Xaritscin) [reply data/json/items/comestibles/meat_dishes.json]: done

File: data/json/items/comestibles/meat_dishes.json [hunk 2]
```diff
@@ -1671,5 +1671,47 @@
...
+    "type": "COMESTIBLE",
+    "id": "escargot",
+    "name": { "str_sp": "escargot" },
+    "weight": "20 g",
+    "color": "brown",
+    "spoils_in": "1 day",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 40,
+    "description": "A traditional dish originated from France, more specifically from the region of Bourgogne.  It consists of prepared snails that have then been cooked in butter and parsley inside their own shells, very low on fat but high on protein.",
```
[the remark this answers is not in the document]
Comment (Xaritscin) [reply data/json/items/comestibles/meat_dishes.json]: done

**Pull request merged**